nip47/event_handler.go

```diff
@@ -134,7 +146,7 @@ func (svc *nip47Service) HandleEvent(ctx context.Context, relay nostrmodels.Rela
 				Message: fmt.Sprintf("Failed to save app to nostr event: %s", err.Error()),
 			},
 		}
-		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss)
+		resp, err := svc.CreateResponse(event, nip47Response, nostr.Tags{}, ss, svc.keys.GetNostrSecretKey())
```

Comment:
This one is definitely wrong. If we can't save events to the DB I think we should log an error and return

Reply:
If we can response, maybe it's better for the client to get an error than get nothing, wdyt?
But yes, the key should be appWalletPrivKey. I'll fix it if you agree we should respond.

Reply:
I don't think there's any point because the client will not listen on a different key. I think just logging an error and returning is enough.

Reply:
I think at that point it already found the app so the client is using a correct key, and we can respond using the `appWalletPrivKey` for that wallet right?

Reply:
oops, yeah I was on the wrong thread. Thanks